app/validation/domain.py

```diff
@@ -65,14 +65,14 @@ def validate_to_from_not_equal(self, data):
             raise ValidationError("msg_to and msg_from fields can not be the same.")
```

Comment:
Should this be consistent with the other ValidationError lines in regards to single quotes rather than double quotes?